### DISCUSSION TITLE
Trying to backup all databases except one database is not working

### DISCUSSION THREAD

**Discussion by jdrobish** [olahallengren/sql-server-maintenance-solution#646] [Questions & Answers] [answered]
I am trying to use the option to backup all databases but excluding one database. I have it coded like this 

EXECUTE [dbo].[DatabaseBackup]
@Databases = 'USER_DATABASES',  -database1
@Directory = NULL,
@BackupType = 'FULL',
@Verify = 'Y',
@CleanupTime = NULL,
@CheckSum = 'Y',
@LogToTable = 'N'

I get an error  saying  Incorrect syntax near 'database1'. so i added a comma after the parameter -database1 and I still get the same error. I'm not sure what else to try.

**Comment by heaivilinn** [comment]
You appear to be missing a single quote ( ‘ ) after the -database1.  Should look something like this:

EXECUTE [dbo].[DatabaseBackup]
@databases = 'USER_DATABASES', -database1’
@Directory = NULL,
@BackupType = 'FULL',
@verify = 'Y',
https://github.com/Cleanuptime = NULL,
@Checksum = 'Y',
@LogToTable = 'N'

Nathan

**Reply by jdrobish** [reply]
I added the single quote suggested and I get the following error:
Message
Executed as user: NT SERVICE\SQLSERVERAGENT. Incorrect syntax near 'database1'. [SQLSTATE 42000] (Error 102)  Unclosed quotation mark after the character string ''. [SQLSTATE 42000] (Error 105).  The step failed.

**Comment by heaivilinn** [comment, marked as answer]
Sorry, mistake in my example.  There is a single quote after USER_DATABASES that should be removed.  The CVS string should be in a single pair of single quotes.

EXECUTE [dbo].[DatabaseBackup]
@databases = 'USER_DATABASES, -database1’
@Directory = NULL,
@BackupType = 'FULL',
@verify = 'Y',
https://github.com/Cleanuptime = NULL,
@Checksum = 'Y',
@LogToTable = 'N'

**Reply by jdrobish** [reply]
that was  it ! Thank you so much!
